extensions/reclaim-ai/src/notifications.tsx

```diff
@@ -154,6 +165,15 @@ export default function Command() {
           };
     }
 
+    if (showNowEvent) {
+      return {
+        fullTitle: "No now events",
+        minTitle: "No now events",
+        nowOrNext: "NONE",
+        event: null,
+      };
+    }
```

Comment:
following on the above comment, I think we're good just leaving this as "No upcoming events" - that text feels appropriate for both now-and-upcoming, or just upcoming.